resources/views/user_home.blade.php

```diff
@@ -0,0 +1,6 @@
+@extends('layouts.app')
```

Comment:
What does this blade file do?

Reply:
I deleted because I was booking with other page.